nixpkgs/nixpkgs.bzl

```diff
@@ -337,6 +337,27 @@ def nixpkgs_packages(
     repository = None,
     nixopts = [],
     ):
+    """
+    Defines a set of targets pointing to nixpkgs packages at once.
+
+    For each `(name, package)` pair in `packages`, this macro will define a
+    `@package` external repository pointing to the nix package `package`
+
```

Comment:
“Internally, this calls `nixpkgs_packages_instantiate` once (meaning only one nix evaluation), then `nixpkgs_package_realize` once for each package.”

Reply:
I'm not font of exposing the internals in the documentation (not that it's a really hidden internal but just because it's an unnecessary burden for the reader). What about "This is equivalent (but faster) to calling several times `nixpkgs_package`" ?